tests/helpers/vitest_setup.ts

```diff
@@ -1,20 +1,6 @@
 import { pathExists } from 'path-exists'
 import { expect } from 'vitest'
 
-interface CustomMatchers {
-  toPathExist(): Promise<void>
-}
-
-declare global {
-  // eslint-disable-next-line @typescript-eslint/no-namespace
-  namespace Vi {
-    // eslint-disable-next-line @typescript-eslint/no-empty-interface
-    interface Assertion extends CustomMatchers {}
-    // eslint-disable-next-line @typescript-eslint/no-empty-interface
-    interface AsymmetricMatchersContaining extends CustomMatchers {}
-  }
-}
```

Comment:
inlined into main.test.ts, as otherwise, this does not apply. I tried different configs, but couldn't get this custom config working.